File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let lookup=new Set();
        for(let num of nums){
            if(lookup.has(num)){
                return true
            }
            lookup.add(num);
        }return false
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        lookup={}
        for i in nums:
            if i not in lookup:
                lookup[i]=0
            lookup[i]+=1
        uni=[key for key,val in lookup.items() if val>=k]

        return uni






                
